### ISSUE TITLE
Update public demo site

### ISSUE THREAD

**Issue by Balearica** [naptha/tesseract.js#768]
The first Google result for "tesseract.js" is the site at [tesseract.projectnaptha.com](https://tesseract.projectnaptha.com/), which has not been updated in years and uses an old version of Tesseract.js.  Additionally, users have reported errors (see #767).  We should update to use the latest version of Tesseract.js.

**Comment by rugk** [comment]
It is also linked in the GitHub repo?
![grafik](https://github.com/naptha/tesseract.js/assets/11966684/554d16b6-bbd4-4d90-bbc9-a1cd0890dde4)

So what is your new public site then?
IMHO, actually, i was just a user searching for a simple "online"/(JS-)offline OCR reader and found this which looks like, but just needs a working demo site?

**Comment by rugk** [comment]
> Tesseract.js you should use the examples in [this folder](https://github.com/naptha/tesseract.js/tree/master/examples/browser),

Uhm you can just use GitHub Pages very simply to host these?

**Comment by rugk** [comment]
It is as simple as going to the settings of this repo and enabling the correct settings (you can AFAIK define the branch and directory to deploy): https://docs.github.com/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

Only thing missing may be a simple `index.html` page to see/link the actual examples in a nice way…

BTW a broken web page is not a good publicity for an otherwise maintained library, because then the whole thing looks quite "broken" from the outside.

**Comment by Balearica** [comment]
If you are an end user looking to OCR a document using Tesseract.js, you can use [scribeocr.com.](https://scribeocr.com/).  This site supports features that we cannot include in our demo sites due to licensing issues (notably, relating to reading and writing PDFs).  You upload a document, click "recognize", and can download the output in multiple formats.  If you choose to download a PDF, be sure to select `View -> Display Mode -> OCR` or you will end up with visible text.

I agree that the demo pages should be fixed, hence the Git Issue.

**Comment by rugk** [comment]
Yeah thanks a lot, that was my main use case (though I am a dev too and find this project very interesting and worth supporting). So maybe that can be linked in the Readme or so, too? I mean you see how a link from GitHub affects search engines ranks and stuff, so… I also just came here searching for "ocr js offline" or something similar and the first thing I saw was your outdated website, which did not work… what i wanted to see/find is exactly that! https://scribeocr.com/
